### ISSUE TITLE
Remove trivial duplication in connected UDP example in `doc/core/howto/udp.xhtml`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4869]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4869|
|Type|enhancement|
|Created|2011-02-11 05:27:58Z|
|Branch|https://github.com/twisted/twisted/tree/udp-sphinx-4869|

The connected UDP example duplicates the host and port information.  It could avoid this.

<details><summary>Searchable metadata</summary>

```
trac-id__4869 4869
type__enhancement enhancement
reporter__exarkun exarkun
priority__low low
milestone__ 
branch__branches_udp_sphinx_4869 branches/udp-sphinx-4869
branch_author__screwtape__jdb screwtape, jdb
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1297402078000000 1297402078000000
changetime__1299005464000000 1299005464000000
version__None None
owner__Screwtape Screwtape

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30670]) Branching to 'udp-sphinx-4869'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30671]) Merge the deferred parts of the impossibly huge [[#4568](https://github.com/twisted/twisted/issues/4568)](https://github.com/twisted/twisted/issues/4568) branch

refs #4568
refs [#4869](https://github.com/twisted/twisted/issues/4869)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to Screwtape|
|-|-|

There don't appear to be any actual Sphinx-related changes in this document.  The conversion output in trunk doesn't have any obvious deficiencies.  The branch just reduces some data duplication in one of the examples, which is fine.  Please add a news fragment (_misc_) I would say and merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [30865]) Merge udp-sphinx-4869 

Author: jdb, screwtape
Reviewer: exarkun
Fixes: [#4869](https://github.com/twisted/twisted/issues/4869)

Clean up some simple duplication in one of the UDP examples.
